packages/vscode/README.md

```diff
@@ -52,6 +55,7 @@ You can find both the [stable](https://open-vsx.org/extension/Prisma/prisma) and
 - Quick-fixes
   - Quickly fix typos in model and enum names
   - Create new models and enums with a single click
+  - For the "Error parsing attribute "@relation": The argument `references` must refer to a unique criteria in the related model. Consider adding an `@unique` attribute to the field `field` in the model `A`." it will add an `@unique` atribute where it's needed.
```

Comment:
```suggestion
  - Add `@unique` on model fields where it is missing for a `@relation`, `references` value
```

Reply:
The end of your sentence is a bit weird 🤔 I don't understand 
```
, `references` value in another model
```

Reply:
It is the `references` value of a `@relation` that is pointing to a field that should have a `@unique`.

Reply:
So...
>Add `@unique` on model fields for a `@relation` where the `references` value is pointing to a field missing it.

Reply:
Not the greatest English, but works for me. It is not the easiest problem to explain.

Reply:
```suggestion
  - Add `@unique` on model fields for a `@relation` where the `references` value is pointing to a field missing it.
```